scripts/gitVersionFile.ts

```diff
@@ -45,32 +45,24 @@ const baseDir = join(_currentPath, '..')
 const versionFileName = 'release-version.json'
 
 async function main() {
-  const cwd = path.join(__dirname, '..')
-  const disklet = makeNodeDisklet(cwd)
+  const cwd = join(__dirname, '..')
   const [branch] = process.argv.slice(2)
 
   // Determine the current version:
-  const packageJson = JSON.parse(await disklet.getText('package.json'))
+  const packageJson = JSON.parse(fs.readFileSync(join(cwd, 'package.json'), { encoding: 'utf8' }))
   const version = `${packageJson.version}${pickVersionSuffix(branch)}`
 
-  const versionFile = getVersionFile(branch, version)
-  if (versionFile == null) throw new Error('Could not get version file')
-
-  console.log(versionFile)
-
-  // Update the native project files:
-  await Promise.all([updateAndroid(disklet, versionFile), updateIos(cwd, versionFile)])
+  getVersionFile(branch, version)
 }
 
-function getVersionFile(branch: string, version: string): VersionFile | undefined {
+function getVersionFile(branch: string, version: string): void {
```

Comment:
Should we rename this function now that it doesn't return the version? `updateVersionFile`?